### DISCUSSION TITLE
Help Request:  issue id : #12679.  problem: Not able to render the login page for student/instructor

### DISCUSSION THREAD

**Discussion by ArunErram** [TEAMMATES/teammates#12729] [Help requests]
### Environment

- **Commit SHA:** 
- **Operating System:**
- **JDK Version:**
- **Node.js Version:**
- **Firefox / Chrome Version (if applicable):**

### Intention

Trying to login as a student/Instructor but after clicking on the login option the login page is not rendering .

**Comment by weiquu** [comment]
Have you followed the steps in the [Developer documentation](https://teammates.github.io/teammates/development.html)? You'll need to start the Docker containers, start the dev server, and start the front end.

If you've done so and still face the issue, please send us the logs so we can take a closer look

**Comment by abdullahsohailcs** [comment]
I encountered a few issues during my project setup that you might find helpful to be aware of:

**Directory Address with Spaces:**
When my directory path contained spaces, such as in "D:\hehe\BS CS\semester7\FYP\teammates", my backend encountered functionality issues. This resulted in errors indicating that certain resources couldn't be found. To address this, I renamed the folder from "BS CS" to "bscs", which resolved the problem partially. Although the backend started working, some features remained dysfunctional.

**Insufficient Disk Space:**
Insufficient disk space can pose a significant obstacle, especially when utilizing Docker. It's crucial to ensure that your disk has adequate space available to prevent any issues.

**Backend Verification:**
Check console/terminal whether backend is working properly or not. Neglecting this step can lead to unexpected behavior in your project.

I hope you find it helpful 😀

**Comment by ArunErram** [comment]
@weiquu  i follwed the steps and i am using these app_admin@gmail.com for these http://localhost:8080/web/instructor/home , but i am getting response as "You are not authorized to view this page.
Log out and return to main page."

and i also used these temates.instructure@university.edu an geting these reposne "Ooops! Your Google account teammates.instructor@university.edu is not known to TEAMMATES".


can you help me out in these please

**Reply by weiquu** [reply]
Hi @ArunErram, can you check your `test.properties` file? It might be that your admin value as set to `app_admin` (instead of `app_admin@gmail.com`). Once you've logged in, you should be able to access the admin pages

**Comment by ArunErram** [comment]
@weiquu  it's `app_admin@gmail.com `in the `test.properties` file. after login , clicking on  drop box of `app_adming@gmail.com`.  i am heading to `To admin pages -> session `. and getting respose are no session found , below are  the snaps .

<img width="835" alt="image" src="https://github.com/TEAMMATES/teammates/assets/140548924/a55e8bb8-d4cc-460c-aeb4-f2094dafb32f">


<img width="868" alt="image" src="https://github.com/TEAMMATES/teammates/assets/140548924/aa267edb-3268-409d-b697-f2fee7f3b998">

**Comment by ArunErram** [comment]
@weiquu , @domlimm can I get any update for the above issue  please?
